name/templates/name/introduction.html

```diff
@@ -0,0 +1,3 @@
+<h3>Introduction</h3>
+<p>The Name App is a tool for documenting names used by the Libraries in its various digital library systems and collections.  The app provides a consistent way of communicating the authorized version of a name and information about the name that is useful for reuse.  The Name App generates a unique URL for each name that can be used to unambiguously refer to a person, organization, event or piece of software.  In addition to an HTML page for each name there are a number of other formats available for each record including a MADS XML version and a simple JSON representation.  A key feature of the Name App is the ability to link to other vocabularies such as the Virtual International Authority File (VIAF), the Library of Congress Name Authority File, or Wikipedia.  As time permits, the Libraries will integrate links into name records where appropriate.  A set of APIs are provided for integration of this application into other systems.  If you have questions or comments about the Name App, or if you would like a name added, please contact the Name App Administrator at <a href="mailto:contact@example.edu">contact@example.edu</a>.</p>
```

Comment:
"unambiguously refer to a person, organization, event or piece of software" should also mention building if we are going to list them all.